ui/core.js

```diff
@@ -157,13 +157,13 @@ $.extend( $.expr[ ":" ], {
 		},
 
 	focusable: function( element ) {
-		return focusable( element, !isNaN( $.attr( element, "tabindex" ) ) );
+		return focusable( element, $.attr( element, "tabindex" ) != null );
```

Comment:
"0xFF" and "Infinity" aren't valid values. We've decided not to account for such markup.